R/cobertura.R

```diff
@@ -32,6 +46,8 @@ to_cobertura <- function(cov, filename = "cobertura.xml"){
 
   # Add sources
   sources <- xml2::xml_add_child(top, "sources")
+  xml2::xml_add_child(sources, "source", xml2::xml_cdata(getwd()))
```

Comment:
This one's a bit of a problem, since I can only discover such a path (or root) attribute in coverage objects from `package_coverage()`. I've now made the add_child conditional on either the package or root path being available, and else only printing an empty node.